routes/api/hacker.js

```diff
@@ -307,6 +307,37 @@ module.exports = {
                 Middleware.Hacker.sendStatusUpdateEmail,
                 Controllers.Hacker.updatedHacker
             );
+        
+        /**
+         * @api {patch} /hacker/acceptEmail/:email accept a Hacker
+         * @apiName acceptHacker
+         * @apiGroup Hacker
+         * @apiVersion 2.0.0
+         *
+         * @apiSuccess {string} message Success message
+         * @apiSuccess {object} data Hacker object
+         * @apiSuccessExample {object} Success-Response:
+         *      {
+         *          "message": "Changed hacker information",
+         *          "data": {
+         *              "status": "Accepted"
+         *          }
+         *      }
+         * @apiPermission Administrator
+         */
+        hackerRouter
+        .route("/acceptEmail/:email")
+        .patch(
+            Middleware.Auth.ensureAuthenticated(),
+            Middleware.Auth.ensureAuthorized([Services.Hacker.findByEmail]),
+            Middleware.Validator.RouteParam.emailValidator,
+            Middleware.parseBody.middleware,
+            Middleware.Hacker.findByEmail,
```

Comment:
is middleware hacker findByEmail and then calling middleware findByEmail duplicate work?

Reply:
Yeah I believe so I'll fix it

Reply:
Edit: req.body.hacker is changed by findByEmail so no it is not duplicated work